src/components/Button/Button.module.css

```diff
@@ -258,7 +253,17 @@
   }
 }
 
-/* The animation code */
+/**
+ * Loading icon rotiation style
+ */
+.button.eds-is-loading > .button__icon {
+  animation: rotateIcon 2s linear infinite; /* 1 */
+  overflow: visible;
```

Comment:
The loading icon was getting cut off and looked jank before adding `overflow: visible`.